php.nanorc

```diff
@@ -30,26 +33,25 @@ color magenta ";"
 color yellow "(<|>)"
 # Assignment operator
 color brightblue "="
-# Bitwise Operations
+# Bitwise operators
 color magenta "(&|\||\^)"
 color magenta "(<<|>>)"
 # Comparison operators
 color yellow "(==|===|!=|<>|!==|<=|>=|<=>)"
 # Logical Operators
-color yellow "( and | or | xor |!|&&|\|\|)"
-# And/Or/SRO/etc
-color cyan "(\;\;|\|\||::|=>|->)"
+color yellow "\<(and|or|xor)\>" "(!|&&|\|\|)"
+# SRO and other operators
+color cyan "(\;\;|::|=>|->)"
 # Double quoted STRINGS!
-color red "(\"[^\"]*\")"
+color red "\"([^"\]|\\.)*\""
+# Single quoted string
+color red "'([^'\]|\\.)*'"
 # Heredoc (typically ends with a semicolon).
-color red start="<<<['\"]?[A-Z][A-Z0-9_]*['\"]?" end="^[A-Z][A-Z0-9_]*;"
+color red start="<<<[\"]?[A-Z][A-Z0-9_]*[\"]?" end="^[A-Z][A-Z0-9_]*;"
```

Comment:
I removed single quote here, because `<<<'EOF'` will been highlighted in line 64.